### PR TITLE
Uricontent: add tests for uricontent

### PR DESCRIPTION
**Pull request by TheKharleeci**
Previous PR: #625 

Uricontent-01: DetectUriSigTest02
Uricontent-02: DetectUriSigTest03
Uricontent-03: DetectUriSigTest05
Uricontent-04: DetectUriSigTest06
Uricontent-05: DetectUriSigTest07
Uricontent-06: DetectUriSigTest08-23 & HttpUriTest01-04

### CODE REVIEW

File: tests/uricontent/uricontent-03/README.md [hunk 1]
```diff
@@ -0,0 +1 @@
+Tests the modifiers for uricontent and content match
```
Comment (inashivb) [inline tests/uricontent/uricontent-03/README.md:1]: This and the following two tests have the same description, do they do nothing differently?

**Review by inashivb — COMMENTED** [review]
LGTM. :) A question inline.

File: tests/uricontent/uricontent-03/README.md (hunk 1 above)
Comment (TheKharleeci) [reply tests/uricontent/uricontent-03/README.md:1]: No. I think I should have converted them to a single test.

**Comment by TheKharleeci** [comment]
Followed By: #659

**Pull request closed without merging**